addresspoints/src/main/resources/application.conf

```diff
@@ -4,6 +4,38 @@ akka {
 	log-dead-letters-during-shutdown = on
 }
 
+kamon {
+  statsd {
+    hostname = "localhost"
```

Comment:
Nevermind, answered my own question on this one:
* http://blog.michaelhamrah.com/2014/02/leveraging-typesafes-config-library-across-environments/